client/lib/application/src/access-form/access-form.jsx

```diff
@@ -82,15 +84,24 @@ class AccessForm extends React.Component {
         });
     }
 
+    fetchScopes(resourceType) {
+      this.props.essentialsActions.fetchScopes(resourceType).then(val => {
+        const state = this.state;
```

Comment:
State should be immutable. Use Object.assign to new state object and change properties there and then set them back to state with .setState(newState);

Reply:
Resolved